routes/products.router.js

```diff
@@ -10,23 +10,29 @@ router.get("/", (req, res) => {
 });
 
 router.post("/", (req, res) => {
-	const sentData = req.body;
+	const receivedData = req.body;
 	const newProduct = new Product({
-		name: sentData.name,
-		price: sentData.price,
-		image: sentData.image,
-		description: sentData.description,
-		ratings: sentData.ratings,
-		delivery: sentData.delivery,
-		offer: sentData.offer,
-		inStock: sentData.inStock,
+		brand: receivedData.brand,
```

Comment:
Instead of writing all the properties separately use spread operator where property name and req body property name is same

Reply:
done